src/applications/check-in/pre-check-in/pages/Introduction/index.jsx

```diff
@@ -24,8 +23,9 @@ const Introduction = props => {
   const dispatchSetVeteranData = useCallback(
     payload => {
       dispatch(setVeteranData({ ...payload }));
+      dispatch(updateFormAction({ ...payload, isEmergencyContactEnabled }));
```

Comment:
@mdewey & @dcgoodwin2112 should this just be a single dispatch that takes care of both? I like the visual representation of what is happening, but two dispatches dealing with the same payload is weird to me.

Reply:
I have been using [batch](https://react-redux.js.org/api/batch) for things like this to handle multiple dispatches in a row. There are some "performance concerns"  and "asynchronous concerns" around multiple dispatches but batch seems to fix that.